lib/auth/saml_test.go

```diff
@@ -130,3 +131,84 @@ func TestEncryptedSAML(t *testing.T) {
 	require.NoError(t, err)
 	require.NotEmpty(t, assertionInfo.Assertions)
 }
+
+func TestPingSAMLWorkaround(t *testing.T) {
```

Comment:
Add comment to test explaining the functionality you're testing.

Reply:
Done, thanks!